### DISCUSSION TITLE
Vote for more Log Drains destinations

### DISCUSSION THREAD

**Discussion by Ziinc** [supabase/supabase#28324] [Feature Requests]
This tracks the demand for different destinations and tooling for log drains. Give a thumbs up if you would like to see it supported.

Each top-level comment should be a unique destination that has not been previously commented.

**Comment by Ziinc** [comment]
Honeycomb

**Comment by Ziinc** [comment]
New Relic

**Comment by Ziinc** [comment]
Splunk

**Comment by Ziinc** [comment]
Betterstack

**Comment by Ziinc** [comment]
Axiom

**Reply by bdsqqq** [reply]
happy to help make it happen from Axiom's side :eyes:

**Reply by Ziinc** [reply]
Hi we've added documentation to help make adding new analytics backends (and by extension log drains) for external contributors:

https://docs.logflare.app/contributing/

So long as there is an adaptor there we can wire things up on the platform

**Comment by Ziinc** [comment]
SigNoz

**Reply by therealpandey** [reply]
Hey @Ziinc,
SigNoz maintainer here! Happy to build this out. I was searching for some documentation on how Supabase sends its payloads but couldn't find anything useful. Could you point me to the right documentation?

I'm basically looking for the structure of the payload.

**Reply by Ziinc** [reply]
Hi we've added documentation to help make adding new analytics backends (and by extension log drains) for external contributors:

https://docs.logflare.app/contributing/

So long as there is an adaptor there we can wire things up on the platform

**Reply by Nageshbansal** [reply]
Hey @Ziinc, SigNoz team here.

We've opened a PR on Logflare to add SigNoz as a backend: https://github.com/Logflare/logflare/pull/3839

Would be great to get your eyes on it when you have a moment.

**Reply by mahakkaur22** [reply]
Thanks for getting the SigNoz backend adaptor merged, @Ziinc !

We’d love to move forward with making SigNoz a native Log Drain destination in Supabase. From our understanding, the next step is enabling signoz as a supported backend type on the Supabase platform side, after which we can proceed with the Studio integration.

Could you let us know what the right next step is to get this enabled, or if there’s someone else on the Supabase team we should coordinate with?

Happy to help with anything needed from the SigNoz side.

**Reply by mahakkaur22** [reply]
Hi @Ziinc, following up on the last note, could you help us get SigNoz added as a native destination in the Supabase Log Drains picker?

Is this something you can wire up on the platform/Studio side, or could you tag the person who owns it? We can provide the configuration fields, icon, and documentation immediately.

**Reply by Nageshbansal** [reply]
Hey @Ziinc, could you please helps us with the next steps and have Signoz as a log drain in Supabase?

**Reply by Ziinc** [reply]
We're stretch on capacity on getting this integrated and tested, patience is appreciated. More log drains work is slated for Q4.

**Comment by Ziinc** [comment]
s3-compatible APIs

**Comment by inian** [comment]
Filebeat

**Comment by inian** [comment]
Loki

**Comment by chasers** [comment]
Clickhouse

**Comment by ggpepp41** [comment]
Sentry

**Reply by AbhiPrasad** [reply]
Hey! I made a PR for this here: https://github.com/Logflare/logflare/pull/2787

Will follow up to hook everything up in https://github.com/supabase/supabase and add docs!

**Comment by martinjt** [comment]
OpenTelemetry

**Comment by ccschmitz** [comment]
Highlight.io

**Reply by Vadman97** [reply]
Happy to help how we can from our end! Already support OTLP and HTTP as destinations.

**Reply by Ziinc** [reply]
Hi we've added documentation to help make adding new analytics backends (and by extension log drains) for external contributors:

https://docs.logflare.app/contributing/

So long as there is an adaptor there we can wire things up on the platform

**Comment by mgm1313** [comment]
[appsignal](https://docs.appsignal.com/logging.html)

**Comment by jonhilgart22** [comment]
logfire

**Comment by brianlmoon** [comment]
HyperDX

**Comment by Ziinc** [comment]
syslog

**Comment by kaspernissen** [comment]
[Dash0](https://www.dash0.com/)

**Comment by pree-dew** [comment]
[Last9](https://last9.io/)

**Reply by sahilk** [reply]
We'll sort out any help you guys need from Last9 to make this happen. :)

**Reply by Ziinc** [reply]
Hi we've added documentation to help make adding new analytics backends (and by extension log drains) for external contributors:

https://docs.logflare.app/contributing/

So long as there is an adaptor there we can wire things up on the platform

**Comment by creator744-ui** [comment]
Please support

**Comment by StefanPostma** [comment]
Dynatrace, But Open telemetry will do!

**Comment by SergeC** [comment]
[Elastic Stack: (ELK) Elasticsearch, Kibana & Logstash](https://www.elastic.co/elastic-stack)

**Comment by m1a8h7sg90i** [comment]
SigNoz

**Comment by sujeet** [comment]
[oodle.ai](https://www.oodle.ai/)

**Reply by sujeet** [reply]
Hi @Ziinc!

Congratulations on the launch of Supabase Log Drains!
Can we get some first-party TLC? 😇

We have updated our docs and our product with support for Supabase:
 - Docs: https://docs.oodle.ai/integrations/logs/supabase
 - Product: https://play.oodle.ai/settings?category=logs&search=supabase

Cheers!

**Comment by gtraversa** [comment]
GCS

**Comment by steamhammercouk** [comment]
**Coralogix**
Cheap long term data storage and a great observability platform on top!

**Comment by tomnagengast** [comment]
Sazabi

**Comment by PhantomWatson** [comment]
[Rollbar](https://rollbar.com/)

**Comment by FrancoisPp** [comment]
Posthog

**Reply by megalath** [reply]
+100

**Comment by Bash4195** [comment]
[AppSignal](https://www.appsignal.com/)

**Comment by kv4royalbase** [comment]
hi am vonny

**Comment by kv4royalbase** [comment]
![Uploading KAPEKE.jpg…]()
